package.json

```diff
@@ -41,6 +41,7 @@
     "semver": "4.3.6",
     "spdlog": "0.6.0",
     "sudo-prompt": "^8.0.0",
+    "twitter-text": "^2.0.4",
```

Comment:
One thing I can't figure out is that typescript keeps throwing `Cannot find module 'twitter-text'` even when I've added the required types too and it does compile properly. Am I missing something?